docs/sources/operators-guide/running-production-environment/planning-capacity.md

```diff
@@ -126,7 +126,10 @@ Estimated required CPU, memory, and disk space:
 ### (Optional) Ruler
 
 The [ruler]({{< relref "../architecture/components/ruler/index.md" >}}) component resources utilization is determined by the number of rules evaluated per second.
-The rules evaluation is computationally equal to queries execution, so the querier resources recommendations apply to ruler too.
+
+When [internal]({{< relref "../architecture/components/ruler/index.md#internal" >}}) mode is enabled, rule evaluation is computationally equal to queries execution, so the querier resources recommendations apply to ruler too.
+
+On the other hand if configured in [remote]({{< relref "../architecture/components/ruler/index.md#internal" >}}) operational mode, most of the computational load will be shifted to query-frontend, query-scheduler, and querier components. So those should be scaled accordingly to deal with the extra workload.
```

Comment:
We can omit query-scheduler because it's a very lightweight process, and basically 2 replicas are enough for almost whatever scale you run at.

```suggestion
When [remote]({{< relref "../architecture/components/ruler/index.md#internal" >}}) operational mode is used, most of the computational load is shifted to query-frontend and querier components. So those should be scaled accordingly to deal both with queries and rules evaluation workload.
```